packages/api/read_test.go

```diff
@@ -91,7 +91,7 @@ func TestRead(t *testing.T) {
 			row = $data[i]
 			if i == 1 || i == 3 {
 				row["my"] = "No name"
-				$data[i] = row
+				//$data[i] = row
```

Comment:
remove it if it's unnecessary

Reply:
This comment is in the test contract. This is a bug which  should be fixed in another  task.